ansible/library/get_feature.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module:         get_feature
+version_added:  "1.0"
+author:         Yutong Zhang (yutongzhang@microsoft.com)
+short_description: Retrieve features status from DUT by 'show feature status' or 'show features' command 
+description:
+    - Retrieve features status from DUT by 'show feature status' or 'show features' command
+    - The retrieved features status will be returned as a dict 
+'''
+
+EXAMPLES = '''
+- name: Get feature status from DUT
+  get_feature:
+'''
+
+# Example of the output
+'''
+The input:
+$ show feature status
+Feature         State            AutoRestart     SetOwner
+--------------  ---------------  --------------  ----------
+bgp             enabled          enabled
+database        always_enabled   always_enabled
+
+The output:
+{
+    'bgp': 'enabled',
+    'database': 'always_enabled'
+}
+'''
+
+class FeatureModule(object):
+    def __init__(self):
+        self.module = AnsibleModule(argument_spec=dict())
+
+    def run(self):
+        """
+            Main method of the class
+        """
+
+        command_list = ['show feature status', 'show features']
+        for cmd in command_list:
+            try:
+                rc, out, err = self.module.run_command(cmd, executable='/bin/bash', use_unsafe_shell=True)
```

Comment:
Why set `use_unsafe_shell=True`?

Reply:
I copy this from `get_interface.py` here. I look at the document, it says `use_unsafe_shell=True` can support pipe. In this case, we don't need to set `use_unsafe_shell=True` here.